src/server.py

```diff
@@ -23,12 +24,31 @@
 app = FastAPI()
 
 
+class PredictionResult(BaseModel):
```

Comment:
Consider adding a timestamp for when the request was made

Reply:
This would still be returned in the JSON payload.

Reply:
✅  `request_timestamp` added to `PredictionResult` object returned by endpoint.